Policy: MTLS upstream policy (#1182)
3scale/APIcast: gateway/src/apicast/policy/upstream_mtls/Readme.md

> # Upstream MTLs policy.

Reviewer: ```suggestion
# Upstream Mutual TLS policy.
```

> This policy enables the MTLS policy per service, so connection to the upstream

Reviewer: ```suggestion
This policy enables the Mutual TLS policy per API, so connection to the upstream
```

API will use the certificates defined in this policy.

## Configuration

> Using certificates Path (For Openshift/Kubernetes secrets)

Reviewer: ```suggestion
Using certificates Path, both for Openshift and Kubernetes secrets.
```

```
{
  "name": "apicast.policy.upstream_mtls",
  "configuration": {
      "certificate": "/secrets/client.cer",
      "certificate_type": "path",
      "certificate_key": "/secrets/client.key",

>       "certificate_key_type": "path"

Reviewer 2: Would be good to document what other options exist apart from "path".

  }
}
```

> ## Things to consider

Reviewer: ```suggestion
## Additional considerations
```

> This policy will overwrite `APICAST_PROXY_HTTPS_CERTIFICATE_KEY` and

Reviewer: ```suggestion
This policy overwrites `APICAST_PROXY_HTTPS_CERTIFICATE_KEY` and
```

> `APICAST_PROXY_HTTPS_CERTIFICATE` values and it'll use the certificates set by

Reviewer: ```suggestion
`APICAST_PROXY_HTTPS_CERTIFICATE` values and it uses the certificates set by
```

> the policy, so those ENV variables will have no effect.

Reviewer: ```suggestion
the policy, so those environment variables will have no effect.
```
